items/src/test/scala/weco/api/items/services/ItemUpdateServiceTest.scala

```diff
@@ -38,30 +38,32 @@ class ItemUpdateServiceTest
   def withSierraItemUpdater[R](
     sierraResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
     contentApiResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
-    time: Int
+    clock: Clock
   )(testWith: TestWith[ItemUpdater, R]): R =
     withActorSystem { implicit actorSystem =>
       withSierraSource(sierraResponses) { sierraSource =>
         val contentApiClient = new MemoryHttpClient(contentApiResponses)
         with HttpGet {
           override val baseUri: Uri = Uri("http://content:9002")
         }
-        val mockTime = ZonedDateTime
-          .of(2024, 4, 24, time, 0, 0, 0, ZoneId.of("Europe/London"))
-        val londonClock = new LondonClock {
-          override def timeInLondon(): ZonedDateTime = mockTime
-        }
 
         testWith(
           new SierraItemUpdater(
             sierraSource,
             new VenueOpeningTimesLookup(contentApiClient),
-            londonClock
+            clock
           )
         )
       }
     }
 
+  def withClock(time: Int = 11): Clock = {
```

Comment:
I would implement this using the `TestWith` pattern like this: https://github.com/wellcomecollection/catalogue-api/blob/5cd4f66dcc362aca2fb60baa13349623d68d2d46/items/src/test/scala/weco/api/items/services/ItemUpdateServiceTest.scala#L36

You could have it take a parseable date time string:

```scala
import import weco.fixtures.TestWith

def withClock[R](
  dateTime: String = "2018-04-29T10:15:30.00Z"
)(testWith: TestWith[Clock, R]): R = {
  val instant = Instant.parse(dateTime)
  val zoneId = ZoneId.of("Europe/London")
  testWith(Clock.fixed(instant, zoneId))
}
```